shared/test/loop_mocha_utils.js

```diff
@@ -85,15 +85,21 @@ var LoopMochaUtils = (function(global, _) {
     });
   };
 
+  SyncThenable.resolve = function(result) {
+    return new SyncThenable(function(resolve) {
+      resolve(result);
+    });
+  };
+
```

Comment:
Please add a comment to each "resolve"  function (both the dynamic and static versions in this file) explaining when each one gets used for clarity.